cloud/spawn.go

```diff
@@ -83,6 +84,15 @@ func (so *SpawnOptions) validate(ctx context.Context, settings *evergreen.Settin
 		return errors.New("public key contains invalid base64 string")
 	}
 
+	if !so.SleepScheduleOptions.IsZero() && !so.NoExpiration {
```

Comment:
Should this check be absorbed into the validate function below?

Reply:
Sure, I moved it down.